build-tests/api-extractor-test-01/dist/internal/api-extractor-test-01.d.ts

```diff
@@ -246,6 +246,8 @@ export declare class TypeReferencesInAedoc {
 
 declare const unexportedCustomSymbol: unique symbol;
 
+export declare const VARIABLE: string;
```

Comment:
Should this have a comment?

Reply:
Good point.

I have fixed this issue in my AE7 branch, but I won't backport the fix to AE6, because in this branch the _getReleaseTagForDeclaration() logic doesn't save the comment that it found.

AE7 will be out soon.

---
In reply to: [234018505](https://github.com/Microsoft/web-build-tools/pull/945#discussion_r234018505) [](ancestors = 234018505)